packages/graph/deployment/graph/Dockerfile

```diff
@@ -4,20 +4,26 @@ ENV RUSTUP_HOME=/usr/local/rustup \
     CARGO_HOME=/usr/local/cargo \
     PATH=:$PATH:/usr/local/cargo/bin
 
-COPY rust-toolchain.toml .
+COPY hash_graph/rust-toolchain.toml .
 RUN apk add --no-cache gcc musl-dev && \
     wget -q -O- https://sh.rustup.rs | sh -s -- -y --default-toolchain none --profile minimal && \
     rustup show
 
 FROM rust AS builder
-WORKDIR /app
-COPY . .
+WORKDIR /usr/local/src/graph
+COPY hash_graph .
 ARG PROFILE=production
 RUN --mount=type=cache,target=/usr/local/cargo/registry \
     --mount=type=cache,target=/usr/local/cargo/git \
     --mount=type=cache,target=/app/target \
     cargo install --path bin/hash_graph --profile $PROFILE --all-features --locked
 
-FROM scratch AS runtime
-COPY --from=builder /usr/local/cargo/bin/hash-graph /
-ENTRYPOINT ["/hash-graph"]
+FROM node:18.7.0-alpine3.16 as runtime
+COPY migrations/ /migrations/
+WORKDIR /migrations
+RUN --mount=type=cache,target=/root/.yarn YARN_CACHE_FOLDER=/root/.yarn yarn install
+
+COPY deployment/graph/migrate_and_run.sh /usr/local/bin/migrate_and_run.sh
```

Comment:
nit and non blocking:
I think `/opt/` would be more appropriate, but I don't feel strongly, just a little shorter than this :)

Reply:
I normally use `/opt` for full programs, i.e. binaries which require more files than only the binary itself. As both files are self-contained I put this in `/usr/local`. I don't really mind 🙂